### DISCUSSION TITLE
Wanted help building from source.

### DISCUSSION THREAD

**Discussion by Cameron990** [stride3d/stride#1662] [Q&A] [answered]
Hello. This is a shot in the dark. I have been trying to build stride from source for going on a week and I am not sure what I am doing wrong. I have all the prerequisites, have tried unistallling and reinstalling vs, clearing the Nuget cache, etc. I follow the instruction procisly and use lfs to clone the repo but it always seems like files are missing. Any help woul be appreciated thanks in advance. These are some of the messages I get. 18>C:\Users\camer\Desktop\stride\samples\Tests\Stride.Samples.Tests.csproj(30,3): warning MSB4011: "C:\Users\camer\Desktop\stride\sources\shared\Stride.NuGetResolver\Stride.NuGetResolver.projitems" cannot be imported again. It was already imported at "C:\Users\camer\Desktop\stride\samples\Tests\Stride.Samples.Tests.csproj (3,3)". This is most likely a build authoring error. This subsequent import will be ignored.
18>CSC : error CS0006: Metadata file 'C:\Users\camer\Desktop\stride\sources\engine\Stride.Assets.Models\obj\Debug\net6.0-windows7.0\ref\Stride.Assets.Models.dll' could not be found
18>CSC : error CS0006: Metadata file 'C:\Users\camer\Desktop\stride\sources\editor\Stride.Assets.Presentation\obj\Debug\net6.0-windows7.0\ref\Stride.Assets.Presentation.dll' could not be found
18>CSC : error CS0006: Metadata file 'C:\Users\camer\Desktop\stride\sources\assets\Stride.Core.Assets.CompilerApp\obj\Debug\net6.0-windows7.0\ref\Stride.Core.Assets.CompilerApp.dll' could not be found
18>CSC : error CS0006: Metadata file 'C:\Users\camer\Desktop\stride\sources\editor\Stride.Editor\obj\Debug\net6.0-windows7.0\ref\Stride.Editor.dll' could not be found
18>CSC : error CS0006: Metadata file 'C:\Users\camer\Desktop\stride\sources\editor\Stride.GameStudio\obj\Debug\net6.0-windows7.0\ref\Stride.GameStudio.dll' could not be found
18>CSC : error CS0006: Metadata file 'C:\Users\camer\Desktop\stride\sources\tools\Stride.SamplesTestServer\obj\Debug\net6.0-windows7.0\ref\Stride.SamplesTestServer.dll' could not be found
18>Done building project "Stride.Samples.Tests.csproj" -- FAILED.
========== Build: 2 succeeded, 16 failed, 48 up-to-date, 0 skipped ==========
========== Build started at 5:12 PM and took 17.502 seconds ==========

**Comment by Doprez** [comment]
Just to go through a short checklist have you:
- Installed the .NET 6 SDK
- selected the https://github.com/stride3d/stride/blob/master/build/Stride.sln as your startup
- downloaded .NET desktop development
- downloaded the FBX SDK

and finally if all are checked have you seen this video from marian? https://www.youtube.com/watch?v=bXSC9o-EaR8 it helped me get started originally.

**Reply by Cameron990** [reply]
Thank you for the help. I was able to get it to build successfully but for some reason It isn't showing in the Stride launcher as an option. I was trying to find where Stride downloads and stores the builds and just move the folder there and build it there to see if that works but it doesn't seem to be in an obvious place. Anyway I at least made progress thank you again. 
![image](https://github.com/stride3d/stride/assets/42922695/36c5b734-b5d8-49bf-9537-56bd85323055)

**Comment by manio143** [comment]
The logs you've published say that one of the test projects failed to compile because another projects it depends on failed to compile. Please find the first project which fails and comment the error message for that one.

**Comment by NicusorN5** [comment, marked as answer]
Are you compiling using a Release configuration, or a Debug configuration? Compiling using a Release configuration works on my machine (despite having the same errors when trying to compile using a debug config)

**Reply by Cameron990** [reply]
Hey ya'll this was the answer, thank you, I had to change it to release then hit (build without debugging) after restarting VS, it started stride and now it appears in the launcher as well. To anyone having problems make sure you have. downloaded the FBX SDK and .NET 6 SDK from the microsoft download page, not just .NET in VS.
